debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/J9BCUtil.java

```diff
@@ -114,6 +118,18 @@
 	 */
 	/* note that methodIndex was not used and was removed */
 	public static void j9bcutil_dumpRomMethod(PrintStream out, J9ROMMethodPointer romMethod, J9ROMClassPointer romClass, long flags, int options) throws CorruptDataException {
+		int vmVersion = getVmVersion();
+		dumpRomMethod(out, romMethod, romClass, flags, options, vmVersion);
+	}
+	
+	private static int getVmVersion() throws CorruptDataException {
+		J9JavaVMPointer vm = J9RASHelper.getVM(DataType.getJ9RASPointer());
```

Comment:
Is this the correct vm? I noticed this was used earlier in `DumpRomClassCommand.java`.